### DISCUSSION TITLE
Monospace font not being applied

### DISCUSSION THREAD

**Discussion by fkollmann** [sqlitecloud/discussions#56] [Bug Report] [answered]
Hi,

from my POV there should be a monospace font "Menlo" applied to studio table view:

![image](https://github.com/user-attachments/assets/57495312-933c-4c23-a740-da2b0af219e0)

However, this is how it looks like for me in Vivaldi (Chromium based) on Linux:

![image](https://github.com/user-attachments/assets/ca2595d4-b86c-42a9-a308-dd44734c6d08)

Is this correct?

Best Regards,
Felix

**Comment by danielebriggi** [comment, marked as answer]
Hi fkollmann, thank you for the bug report, we found the issue and we push it in production in the next release

**Comment by fkollmann** [comment]
Looks fine now:

![image](https://github.com/user-attachments/assets/b28386a5-11c4-4172-afc6-5ff07e9748bb)

Thanks for fixing!
